docs/en_US/Release.md

```diff
@@ -1,5 +1,31 @@
 # ChangeLog
 
+## Release 0.9 - 7/1/2019
+
+### Major Features
+* General NAS programming interface
+    * Add `enas-mode` and `oneshot-mode` for NAS interface: [PR #1201](https://github.com/microsoft/nni/pull/1201#issue-291094510)
+* [Gaussian Process Tuner with Matern kernel](./GPTuner.md) 
+
+* Multiphase interface
+    * Support for OpenPAI Training Service
+    * Enable all builtin tuners as multiphase tuner. For details, please refer to [Write a tuner that leverages multi-phase](./MultiPhase.md#write-a-tuner-that-leverages-multi-phase)
+
+* Web Portal
+    * Enable trial comparation in Web Portal 
+    * Allow users to adjust rendering interval of Web Portal. For details, refer to [Issue #1106](https://github.com/microsoft/nni/issues/1106#issuecomment-496343367)
+    * show intermediate results more friendly 
+* [Commandline Interface](Nnictl.md)
+    * `nnictl experiment delete`: delete data for one or more experiments.
+    * `nnictl platform clean`: clean disk on target training service platform.
+
```

Comment:
* `nnictl platform clean [Options]`: It uses to clean up disk on a target platform. The provided YAML file includes the information of target platform, and it follows the same schema as the NNI configuration file.

Reply:
updated in https://github.com/microsoft/nni/pull/1232/commits/030965f6e1acb517d4308188fa3ab6a7bf2e4c69